libs/api/prisma/migrations/20250519153015_add_archived_at_revisions/migration.sql

```diff
@@ -0,0 +1,27 @@
+-- AlterTable
+ALTER TABLE "articles.revisions" ADD COLUMN     "archivedAt" TIMESTAMP(3);
+UPDATE "articles.revisions" SET "archivedAt" = CURRENT_TIMESTAMP WHERE "publishedAt" IS NULL;
+
+WITH latest AS (
+    SELECT id
+    FROM "articles.revisions"
+    ORDER BY "createdAt" DESC
+    LIMIT 1
+)
+UPDATE "articles.revisions"
+SET "archivedAt" = NULL
+WHERE id IN (SELECT id FROM latest) AND "publishedAt" IS NULL;
```

Comment:
check for `publishedAt` here to make sure only the latest revision is affected. If we check it in the `WITH` above, it would affect the latest `draft`, even if there should be no more draft revision